blogposts/announcing-sourcegraph-3.5.md

```diff
@@ -0,0 +1,228 @@
+
+```
+lang:python ^\s*@.*\n\s*def
+```
+
+**Note:** multi-line search is currently only supported on indexed repositories (and, inherently, default branches). We’re [working to improve this](https://github.com/sourcegraph/sourcegraph/issues/4518).
+
+## Restrict search results to repositories with recent activity
+
+When you’re searching over 10,000+ repositories in an enterprise setting, it’s common to get results from old repositories with no active maintainers. You can now exclude repositories with no recent commits (to their default branch) with the beta `repohascommitafter` [search filter](https://docs.sourcegraph.com/user/search/queries).
+
+For example: Find Dockerfiles requiring upgrades from an obsolete version of Debian so you can contact the maintainers:
+
+```
+file:Dockerfile FROM debian\:jessie|8 repohascommitafter:”1 month ago” 
+```
+
+Another example: Find Terraform files with an `aws_instance` resource to analyze EBS volume configuration:
+
+```
+aws_instance lang:hcl repohascommitafter:"1 month ago"
+```
+
+**Note:** `repohascommitafter` is considered beta as performance degrades significantly on very large repository sets. We are [addressing this in 3.6](https://github.com/sourcegraph/sourcegraph/issues/4614).
+
+## Restrict search to repositories with or without a file
+
+Sometimes, you want to search repositories based on the existence or absence of a file. This is now possible with the new `repohasfile` [search filter](https://docs.sourcegraph.com/user/search/queries) which comes in two forms, `repohasfile` and `-repohasfile`.
+
+An example using `repohasfile`, is finding applications with a Dockerfile, written in Go, that use Travis CI:
+
+```
+repohasfile:Dockerfile file:\.travis.yml "language: go"
+```
+
+An example of using `-repohasfile` is finding applications with a Dockerfile that do not contain a `.dockerignore` file:
+
+```
+-repohasfile:\.dockerignore file:Dockerfile
+```
+
+Another, is finding Python applications using `pytest` without a pytest.ini or pytest.conf file:
+
+```
+-repohasfile:(pytest\.ini|pytest\.conf) lang:python "import pytest"
+```
+
+## Powering code alerts with saved searches
+
+<p class="container">
+<div style="padding:56.25% 0 0 0;position:relative;">
+  <iframe src="https://player.vimeo.com/video/342111852?color=0CB6F4&amp;title=0&amp;byline=" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+</div>
+<p style="text-align: center"><a href="https://vimeo.com/342111852" target="_blank">View on Vimeo</a></p>
+</p>
+
+[Saved searches](https://docs.sourcegraph.com/user/search/saved_searches) can act as bookmarks to frequently used searches, and when combined with email notifications, can help monitor critical parts of your code, organization wide.
+
+Saving a search is faster and easier, and is now accessed from the user and organization settings page.
```

Comment:
This also sounds run-on and awkward ("Saving a search is now accessed ..."). It is also not accurate to say they live in the settings pages, because that's the page with the Monaco editor.

```suggestion
User-level and organization-level saved searches are now separate. You can view and manage saved searches in the user and organization profile areas, respectively.
```